frontend/src/app/Settings/Users/ManageUsers.scss

```diff
@@ -0,0 +1,35 @@
+.users-sidenav {
+  border-right: 1px solid #dfe1e2; // color("text-base-lighter");
+}
+
+.users-sidenav-item:hover {
+  cursor: pointer;
+}
+.user-header {
+  border-bottom: 1px solid #dfe1e2;
+  padding: 0em 1em;
+}
+
+.user-content {
+  padding: 0em 1em;
+}
+
+.reset-button {
+  display: block;
+  margin: 1em auto;
+}
+
+.usa-current {
+  .sidenav-user-name {
+    color: #005ea2;
+  }
+}
+
+.sidenav-user-name {
+  color: #171717; // color($theme-color-prime-gray-light)
```

Comment:
iunno how to get variable references to work

Reply:
I tried adding `@import "../app/Settings/Users/ManageUsers.scss"` in App.scss and then recompiled the css. Still doesn't detect the variables tho

Reply:
It should be possible to `@import` just the USWDS global macros, variables, and utilities at the top of each scss module file. It doesn't matter if that's included from multiple files, as long as the file with that stuff doesn't have any css rules.  We need to take the `@import`s at the top of App.scss and make them into a `settings.scss` that everything can include. At least that's the way I remember it. I'd punted on trying to refactor all that and used hard-coded stuff the way you are here. We're slowly whittling away at the global styles though so maybe it's not as bad to clean up as it was a couple of months ago.